src/main/edu/colostate/cs/cs414/method_men/jungle/client/gui/Client.java

```diff
@@ -37,7 +37,7 @@ public static void main(String[] args) throws Exception{
         Thread t2 = new Thread(client, "ClientSend");
         t1.start();
         t2.start();
-        GUI g = new GUI();
+        GUI g = new GUI(client.socket);
         g.startGUI(g, client.socket);
```

Comment:
```suggestion
        g.startGUI(g);
```
The GUI object already has the socket. We don't need to pass it in again.